src/app/successmessage/successmessage.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+
+class success{
```

Comment:
there is no need for this class definition here unless you were just practicing classes